### DISCUSSION TITLE
Skip `coveo auth:login` temporarily

### DISCUSSION THREAD

**Discussion by akaustav** [coveo/cli#245] [Q&A] [answered]
Hi,
We are having some issues associating my Office 365 account to an already existing coveo organization setup for our company. While that is being resolved, I am unable to move forward checking the cli functionality. eg. to generate a sample Angular app and play around with it. Is there a way to temporarily skip the `coveo auth:login` process and build a generic Angular application using [`coveo ui:create:angular NAME`](https://github.com/coveo/cli/tree/master/packages/cli#coveo-uicreateangular-name) command?

Ameet

**Comment by louis-bompart** [comment, marked as answer]
Hey @akaustav!
Thanks for contacting us!
You need to be connected to a Coveo Organization to be able to run any `ui:create` command because we also generate an API Key so that when you try your project locally, you're able to actually have search results.

So, given that you cannot use your Office 365 login, I recommend that you check https://levelup.coveo.com/
By signing in here, you should be able to setup a Developement Organization (see [here](https://levelup.coveo.com/user/devorgs)).

Then, you should be able to do the `coveo auth:login` on this org by using `coveo auth:login -o=YourDevelopmentOrganizationId`.

You can get the Organization Id of your Development Organization by clicking on it on the dev org page of LevelUp.
![image](https://user-images.githubusercontent.com/12366410/120675436-1058e600-c463-11eb-967a-0863ab66ad6a.png)

You'll then have it written here:
![image](https://user-images.githubusercontent.com/12366410/120675553-2cf51e00-c463-11eb-9523-c104f9d97717.png)

Don't hesitate to reach out if you need more assistance!
